### DISCUSSION TITLE
When clicking on tab the font color becomes black, any way to override that? :focus doesn't do anything

### DISCUSSION THREAD

**Discussion by Laestry** [carbon-design-system/carbon-components-svelte#1712] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/49237501/228984090-873b48c8-6b01-433f-84fa-c976b2be09a7.png)

I've tried but doesn't affect the color when I'm clicking on a tab. Is there a way I can change it only for this specific tab group?
`   :global( .bx--tabs__nav-link:focus){
        color: white !important;
    }`

**Comment by theetrain** [comment, marked as answer]
Hello, which theme is this, and which version of `carbon-components-svelte` are you using?

I couldn't reproduce this black-text-on-black-background issue using the `g100` theme: https://svelte.dev/repl/605067b8b1cf4446a367752cfdaf30c6?version=3.58.0
